menubar.sh

```diff
@@ -1,15 +1,47 @@
 #!/bin/bash
 
+OS=$(uname -s)
 SERVICE=growler.py
-OUTFILE=~/Library/Logs/SecurityGrowler.log
+OUTFILE=/tmp/SecurityGrowler.log
```

Comment:
I don't feel like logging in there, I have the suspicion that it would just clutter the space.

AFAIK logging to `/{.local,usr}/share` is not a common practice, but I may just be not informed enough.